### PR TITLE
more ApexDoc for Recurring Donations.

### PR DESCRIPTION
**Pull request by davidhabib**

### CODE REVIEW

**Comment by davidhabib** [comment]
@njjc This pull request is ready for review and merge. thanks.

File: src/classes/RD_AddDonationsBTN_CTRL.cls [hunk 1]
```diff
@@ -30,13 +30,26 @@
 /**
 * @author Salesforce.com Foundation
 * @date 2012 (2.2)
+* @group Recurring Donations
```
Comment (njjc) [inline src/classes/RD_AddDonationsBTN_CTRL.cls]: We need a group-content tag as well.

File: src/classes/RD_RecurringDonations.cls [hunk 2]
```diff
@@ -30,6 +30,10 @@
 /**
 * @author Salesforce.com Foundation
 * @date 2012 (2.0)
+*
```
Comment (njjc) [inline src/classes/RD_RecurringDonations.cls]: I've been not leaving any empty lines here, I think we should follow that for consistency.

File: src/classes/RD_RecurringDonations_TEST.cls [hunk 3]
```diff
@@ -1741,8 +1741,10 @@ public class RD_RecurringDonations_TEST {
         controller.saveCISettings();
     }
 *************************************************************************/
-   
-   // from RD_RecurringDonations_SCHED  
+
+   /**
+   * @description from RD_RecurringDonations_SCHED
```
Comment (njjc) [inline src/classes/RD_RecurringDonations_TEST.cls]: This description seems incomplete.

**Comment by njjc** [comment]
@davidhabib made some notes, let me know when they're fixed and I'll merge!

File: src/classes/RD_AddDonationsBTN_CTRL.cls (hunk 1 above)
Comment (davidhabib) [inline src/classes/RD_AddDonationsBTN_CTRL.cls]: only one file in the group should include the group content tag, and I believe I have it in RD_RecurringDonations.cls

Comment (ceiroa) [inline src/classes/RD_AddDonationsBTN_CTRL.cls]: "Should" or "needs to"? It's ok if more than one has it, right?

**Comment by davidhabib** [comment]
@njjc items addressed.  ready for merge.  build is rerunning.

File: src/classes/RD_AddDonationsBTN_CTRL.cls (hunk 1 above)
Comment (davidhabib) [inline src/classes/RD_AddDonationsBTN_CTRL.cls]: if you have multiple, probably the last one parsed wins.  so it's a "should".

Comment (njjc) [inline src/classes/RD_AddDonationsBTN_CTRL.cls]: Interesting, I wasn't aware of that.

**Pull request merged**